0020-bluetooth-payment-protocol/0020-bluetooth-payment-protocol.md

```diff
@@ -0,0 +1,110 @@
+# Bluetooth Payment Protocol
+
+This document specifies a protocol for communicating and authorizing Interledger Payment Requests over Bluetooth (Low Energy). It is primarily intended for use in in-person retail payment contexts.
+
+This protocol may be implemented by Customer devices such as smartphones or very low-power, offline devices such as [tapFi](https://github.com/joaopedrovbs/tapFi).
+
+Once a Customer becomes active by broadcasting it's payment service over BLE, Merchants can connect
+and request signature for payments over a secure Bluetooth channel. The channel is closed once
+a payment is signed or the Customer cancels the purshase for security or authentication reasons.
+
+## Definitions
+
+- `IPR`: [Interledger Payment Request](https://github.com/interledger/rfcs/blob/master/0011-interledger-payment-request/0011-interledger-payment-request.md)
+- `Merchant`: The one requesting a payment
```

Comment:
We have been using the full link to the `.md` files because it's way better if you're browsing the specs on github. @justmoon's comment was that we should just change the way it's rendered on the website to make sure that works